### PR TITLE
refactor: es6 modules & testable workshop

### PR DESCRIPTION
**Pull request by weichweich**
* switch from `require` to `import`
* make the workshop testable
  * execution will be more linear, less branching

## Checklist:

- [ ] I have verified that the code works
- [ ] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [ ] I have [left the code in a better state](https://deviq.com/principles/boy-scout-rule)
- [ ] I have documented the changes (where applicable)

### CODE REVIEW

File: docs/sdk/1-workshop/03-attester/02-did.md [hunk 1]
```diff
@@ -68,53 +65,22 @@ Create a file `attester/generateKeypairs.js` and copy the code below.
   {GenerateKeypairs}
 </CodeBlock>
 
-Now we have all we need to create our `DID`: a `keystore`, `keypairs` and `account`. Create
-a file `attester/createFullDid.js` and paster the following code.
+Once we have created all the necessary keys for a DID we can create the DID on chain.
+To create a DID we first initialize everything as we always do.
```
Comment (Dudleyneedham) [inline docs/sdk/1-workshop/03-attester/02-did.md]:
```suggestion
To create a DID we first initialize everything.
```

Comment (Dudleyneedham) [reply docs/sdk/1-workshop/03-attester/02-did.md]: If you want to add something here, you should add a reason why we need to do this.

File: docs/sdk/1-workshop/04-claimer/01-did.md [hunk 2]
```diff
@@ -18,50 +17,40 @@ Remember light DIDs can:
 
 Take a look at our [DID documentation](/docs/sdk/core-feature/did) if you want to learn more about DIDs and the difference between their light and full versions.
 
-## generateKeypairs
+## Generate Keys
 
-As before we'll first create our signing and encryption `keypairs`. We'll use an inbound keystore to generate them.
+Similar to the attester, the claimer too has to setup the DID keys.
```
Comment (Dudleyneedham) [inline docs/sdk/1-workshop/04-claimer/01-did.md]:
```suggestion
Similar to the attester, the claimer must set up the DID keys.
```

File: docs/sdk/1-workshop/6-verification.md [hunk 3]
```diff
@@ -4,68 +4,58 @@ title: 🤝 Verification
...
 
 :::
 
-## Verifier Index
+:::info Presentation
 
-Let's create our `verifier/index.js`. Here we'll expose `getChallenge` which returns a random and unique
-challenge for the  <span class="label-role claimer">Claimer</span> to sign, this is used to prove ownership. We'll also expose `verifyCredential`
-which will do the actual verification. Copy the code below, this completes the  <span class="label-role verifier">Verifier</span> code!
+A `Presentation` object is created by the <span class="label-role claimer">claimers</span> using a credential.
```
Comment (Dudleyneedham) [inline docs/sdk/1-workshop/6-verification.md]: This doesn't makes sense. The claimer creates a credential from what?

**Review by Dudleyneedham — CHANGES_REQUESTED** [review]
Minor suggestions and comments

File: docs/sdk/1-workshop/6-verification.md (hunk 3 above)
Comment (weichweich) [reply docs/sdk/1-workshop/6-verification.md]: The presentation is derived from the credential. The user already has the credential?

Comment (Dudleyneedham) [reply docs/sdk/1-workshop/6-verification.md:25]: But that assumes the user knows what a presentation is and what they make it from. Being explicit is much better

Comment (Dudleyneedham) [reply docs/sdk/1-workshop/6-verification.md]:
```suggestion
A credential creates the `Presentation` object by the <span class="label-role claimer">claimers</span>.
```

Comment (Dudleyneedham) [reply docs/sdk/1-workshop/6-verification.md]: The concept is fine, but I mean the way it reads.

File: docs/sdk/1-workshop/6-verification.md [hunk 4]
```diff
@@ -4,68 +4,58 @@ title: 🤝 Verification
...
 
 :::
 
-## Verifier Index
+:::info Presentation
 
-Let's create our `verifier/index.js`. Here we'll expose `getChallenge` which returns a random and unique
-challenge for the  <span class="label-role claimer">Claimer</span> to sign, this is used to prove ownership. We'll also expose `verifyCredential`
-which will do the actual verification. Copy the code below, this completes the  <span class="label-role verifier">Verifier</span> code!
+A `Presentation` object is created by the <span class="label-role claimer">claimer</span> using a credential.
```
Comment (Dudleyneedham) [inline docs/sdk/1-workshop/6-verification.md]:
```suggestion
A credential is used to create the `Presentation` object by the <span class="label-role claimer">claimer</span>.
```

**Review by Dudleyneedham — APPROVED** [review]
LGTM

**Pull request merged**